packages/inter-protocol/src/psm/psm.js

```diff
@@ -399,6 +399,12 @@ export const start = async (zcf, privateArgs, baggage) => {
         );
       },
       ...publicMixin,
+      getPublicTopics() {
```

Comment:
I could return another helper from `handleParamGovernance` that would build `getPublicTopics()` from something provided by the caller, so `getPublicTopics()` could just be included in `publicMixin`.  Would that be better?